checker/tests/nullness/Issue2432b.java

```diff
@@ -0,0 +1,42 @@
+// Additional test cases for issue 2432 (unrelated to poly annotations)
+// https://github.com/typetools/checker-framework/issues/2432
+
+import java.util.ArrayList;
```

Comment:
The test fails:

```
tests.NullnessFbcTest > run[nullness] FAILED
    java.lang.AssertionError: 790 out of 790 expected diagnostics were found.
    3 unexpected diagnostics were found:
    Issue2432b.java:12: error: (assignment.type.incompatible)
    Issue2432b.java:19: error: (assignment.type.incompatible)
    Issue2432b.java:35: error: (new.array.type.invalid)
    While type-checking /__w/1/s/checker/tests/nullness/MultiAnnotations.java, /__w/1/s/checker/tests/nullness/FlowNullness.java, /__w/1/s/checker/tests/nullness/Issue2470.java, /__w/1/s/checker/tests/nullness/AssertIfFalseTest.java, /__w/1/s/checker/tests/nullness/TestValOf.java, /__w/1/s/checker/tests/nullness/SequenceAndIndices.java, /__w/1/s/checker/tests/nullness/ThisTest.java, /__w/1/s/checker/tests/nullness/ReadyReadLine.java, /__w/1/s/checker/tests/nullness/AssertInStatic.java, /__w/1/s/checker/tests/nullness/JPanelTest.java, /__w/1/s/checker/tests/nullness/RawTypesBounded.java, /__w/1/s/checker/tests/nullness/SuppressWarningsTest.java, /__w/1/s/checker/tests/nullness/Issue2888.java, /__w/1/s/checker/tests/nullness/JavaCopExplosion.java, /__w/1/s/checker/tests/nullness/SuperConstructorInit.java, /__w/1/s/checker/tests/nullness/GeneralATFStore.java, /__w/1/s/checker/tests/nullness/EqualToNullness.java, /__w/1/s/checker/tests/nullness/ForEachMin.java, /__w/1/s/checker/tests/nullness/Issue328.java, /__w/1/s/checker/tests/nullness/Bug102.java, /__w/1/s/checker/tests/nullness/BoxingNullness.java, /__w/1/s/checker/tests/nullness/AssertIfClient.java, /__w/1/s/checker/tests/nullness/Issue293.java, /__w/1/s/checker/tests/nullness/Issue1797.java, /__w/1/s/checker/tests/nullness/Imports1.java, /__w/1/s/checker/tests/nullness/JavaExprContext.java, /__w/1/s/checker/tests/nullness/TernaryNullness.java, /__w/1/s/checker/tests/nullness/Issue2048.java, /__w/1/s/checker/tests/nullness/AssertParameterNullness.java, /__w/1/s/checker/tests/nullness/RawField.java, /__w/1/s/checker/tests/nullness/AssertAfter.java, /__w/1/s/checker/tests/nullness/Issue1983.java, /__w/1/s/checker/tests/nullness/Issue2619.java, /__w/1/s/checker/tests/nullness/LogicOperations.java, /__w/1/s/checker/tests/nullness/NullnessFieldInvar.java, /__w/1/s/checker/tests/nullness/KeyForFlow.java, /__w/1/s/checker/tests/nullness/KeyForSubst.java, /__w/1/s/checker/tests/nullness/Issue580.java, /__w/1/s/checker/tests/nullness/AssertIfNonNullTest.java, /__w/1/s/checker/tests/nullness/EnsuresNonNullIfInheritedTest.java, /__w/1/s/checker/tests/nullness/Issue1922.java, /__w/1/s/checker/tests/nullness/Issue308.java, /__w/1/s/checker/tests/nullness/Issue2052.java, /__w/1/s/checker/tests/nullness/DaikonTests.java, /__w/1/s/checker/tests/nullness/PureTest.java, /__w/1/s/checker/tests/nullness/TernaryNested.java, /__w/1/s/checker/tests/nullness/Synchronization.java, /__w/1/s/checker/tests/nullness/Marino.java, /__w/1/s/checker/tests/nullness/FlowField.java, /__w/1/s/checker/tests/nullness/PrimitivesNullness.java, /__w/1/s/checker/tests/nullness/UnannoPrimitives.java, /__w/1/s/checker/tests/nullness/FlowNonThis.java, /__w/1/s/checker/tests/nullness/PrivateMethodUnknownInit.java, /__w/1/s/checker/tests/nullness/Issue1949.java, /__w/1/s/checker/tests/nullness/NewObjectNonNull.java, /__w/1/s/checker/tests/nullness/InferListParam.java, /__w/1/s/checker/tests/nullness/HasInnerClass.java, /__w/1/s/checker/tests/nullness/LogRecordTest.java, /__w/1/s/checker/tests/nullness/MultipleErrors.java, /__w/1/s/checker/tests/nullness/EnsuresNonNullIfTestSimple.java, /__w/1/s/checker/tests/nullness/Issue331.java, /__w/1/s/checker/tests/nullness/GetProperty.java, /__w/1/s/checker/tests/nullness/GetRefArg.java, /__w/1/s/checker/tests/nullness/AssertTwice.java, /__w/1/s/checker/tests/nullness/Issue1096.java, /__w/1/s/checker/tests/nullness/Conditions.java, /__w/1/s/checker/tests/nullness/KeyForLocalVariable.java, /__w/1/s/checker/tests/nullness/TestAssignment.java, /__w/1/s/checker/tests/nullness/Throwing.java, /__w/1/s/checker/tests/nullness/Issue273.java, /__w/1/s/checker/tests/nullness/DotClass.java, /__w/1/s/checker/tests/nullness/FlowConditions.java, /__w/1/s/checker/tests/nullness/KeyForIssue328.java, /__w/1/s/checker/tests/nullness/VoidUse.java, /__w/1/s/checker/tests/nullness/MethodInvocation.java, /__w/1/s/checker/tests/nullness/Aliasing.java, /__w/1/s/checker/tests/nullness/FinalFields.java, /__w/1/s/checker/tests/nullness/KeyForDiamond.java, /__w/1/s/checker/tests/nullness/LubTest.java, /__w/1/s/checker/tests/nullness/WeakIdentityPair.java, /__w/1/s/checker/tests/nullness/Issue2031.java, /__w/1/s/checker/tests/nullness/OverrideANNA3.java, /__w/1/s/checker/tests/nullness/ChainAssignment.java, /__w/1/s/checker/tests/nullness/VarargsNullness.java, /__w/1/s/checker/tests/nullness/CastTypeVariable.java, /__w/1/s/checker/tests/nullness/HierarchicalInit.java, /__w/1/s/checker/tests/nullness/RequiresNonNullTest.java, /__w/1/s/checker/tests/nullness/NewNullable.java, /__w/1/s/checker/tests/nullness/Issue906.java, /__w/1/s/checker/tests/nullness/Issue1027.java, /__w/1/s/checker/tests/nullness/RawParameter.java, /__w/1/s/checker/tests/nullness/UnboxConditions.java, /__w/1/s/checker/tests/nullness/ThisIsNN.java, /__w/1/s/checker/tests/nullness/UnexpectedRaw.java, /__w/1/s/checker/tests/nullness/Issue752.java, /__w/1/s/checker/tests/nullness/SuppressDeprecation.java, /__w/1/s/checker/tests/nullness/Issue563.java, /__w/1/s/checker/tests/nullness/InfiniteLoopIsSameType.java, /__w/1/s/checker/tests/nullness/ValidType.java, /__w/1/s/checker/tests/nullness/RawTypesUses.java, /__w/1/s/checker/tests/nullness/Issue1628.java, /__w/1/s/checker/tests/nullness/FullyQualifiedAnnotation.java, /__w/1/s/checker/tests/nullness/RawAndPrimitive.java, /__w/1/s/checker/tests/nullness/Issue989.java, /__w/1/s/checker/tests/nullness/TwoStaticInitBlocks.java, /__w/1/s/checker/tests/nullness/MapGetNullable.java, /__w/1/s/checker/tests/nullness/InitSuppressWarnings.java, /__w/1/s/checker/tests/nullness/AssertAfterChecked.java, /__w/1/s/checker/tests/nullness/ToArrayNullness.java, /__w/1/s/checker/tests/nullness/FlowNegation.java, /__w/1/s/checker/tests/nullness/OverrideNNOE2.java, /__w/1/s/checker/tests/nullness/PackageDecl.java, /__w/1/s/checker/tests/nullness/Issue369.java, /__w/1/s/checker/tests/nullness/Issue1522.java, /__w/1/s/checker/tests/nullness/KeyForPolymorphism.java, /__w/1/s/checker/tests/nullness/NonNullInitialization.java, /__w/1/s/checker/tests/nullness/ThisLiteral.java, /__w/1/s/checker/tests/nullness/OverrideANNA2.java, /__w/1/s/checker/tests/nullness/MethodTypeVars4.java, /__w/1/s/checker/tests/nullness/ArrayArgs.java, /__w/1/s/checker/tests/nullness/LazyInitialization.java, /__w/1/s/checker/tests/nullness/EnsuresNonNullIfTest2.java, /__w/1/s/checker/tests/nullness/Wildcards.java, /__w/1/s/checker/tests/nullness/Wellformed.java, /__w/1/s/checker/tests/nullness/Imports2.java, /__w/1/s/checker/tests/nullness/VarInfoName.java, /__w/1/s/checker/tests/nullness/MultiConstructorInit.java, /__w/1/s/checker/tests/nullness/Issue1712.java, /__w/1/s/checker/tests/nullness/Issue471.java, /__w/1/s/checker/tests/nullness/InitializedField.java, /__w/1/s/checker/tests/nullness/FlowConstructor2.java, /__w/1/s/checker/tests/nullness/AssertNullable.java, /__w/1/s/checker/tests/nullness/Issue1102.java, /__w/1/s/checker/tests/nullness/Raw2.java, /__w/1/s/checker/tests/nullness/Enums.java, /__w/1/s/checker/tests/nullness/Issue1307.java, /__w/1/s/checker/tests/nullness/ArrayAssignmentFlow.java, /__w/1/s/checker/tests/nullness/IteratorEarlyExit.java, /__w/1/s/checker/tests/nullness/KeyFors.java, /__w/1/s/checker/tests/nullness/BinaryOp.java, /__w/1/s/checker/tests/nullness/SAMLineParser.java, /__w/1/s/checker/tests/nullness/PostconditionBug.java, /__w/1/s/checker/tests/nullness/MonotonicNonNullTest.java, /__w/1/s/checker/tests/nullness/Widening.java, /__w/1/s/checker/tests/nullness/ArrayRefs.java, /__w/1/s/checker/tests/nullness/RefineArray.java, /__w/1/s/checker/tests/nullness/Issue2171.java, /__w/1/s/checker/tests/nullness/Issue2432.java, /__w/1/s/checker/tests/nullness/KeyForValidation.java, /__w/1/s/checker/tests/nullness/InitializationAssertionFailure.java, /__w/1/s/checker/tests/nullness/JavaCopRandomTests.java, /__w/1/s/checker/tests/nullness/ConditionalNullness.java, /__w/1/s/checker/tests/nullness/NullableArrays.java, /__w/1/s/checker/tests/nullness/Lazy.java, /__w/1/s/checker/tests/nullness/Issue829.java, /__w/1/s/checker/tests/nullness/WeakRef.java, /__w/1/s/checker/tests/nullness/RawInt.java, /__w/1/s/checker/tests/nullness/NNOEMoreTests.java, /__w/1/s/checker/tests/nullness/DefaultLoops.java, /__w/1/s/checker/tests/nullness/Issue391.java, /__w/1/s/checker/tests/nullness/UnannoPrimitivesDefaults.java, /__w/1/s/checker/tests/nullness/WhileTest.java, /__w/1/s/checker/tests/nullness/Polymorphism.java, /__w/1/s/checker/tests/nullness/Issue672.java, /__w/1/s/checker/tests/nullness/FlowInitialization.java, /__w/1/s/checker/tests/nullness/MyException.java, /__w/1/s/checker/tests/nullness/FieldInit.java, /__w/1/s/checker/tests/nullness/WildcardSubtype.java, /__w/1/s/checker/tests/nullness/Issue1059.java, /__w/1/s/checker/tests/nullness/TypeVarPrimitivesNullness.java, /__w/1/s/checker/tests/nullness/CopyOfArray.java, /__w/1/s/checker/tests/nullness/Issue1046Java7.java, /__w/1/s/checker/tests/nullness/RawTypesNullness.java, /__w/1/s/checker/tests/nullness/FindBugs.java, /__w/1/s/checker/tests/nullness/Issue2432b.java, /__w/1/s/checker/tests/nullness/ControlFlow.java, /__w/1/s/checker/tests/nullness/InferTypeArgsCondtionalExpression.java, /__w/1/s/checker/tests/nullness/RawMethodInvocation.java, /__w/1/s/checker/tests/nullness/ExplictTypeVarAnnos.java, /__w/1/s/checker/tests/nullness/ExceptionParam.java, /__w/1/s/checker/tests/nullness/Issue1847.java, /__w/1/s/checker/tests/nullness/Issue338.java, /__w/1/s/checker/tests/nullness/CallSuper.java, /__w/1/s/checker/tests/nullness/AssignmentDuringInitialization.java, /__w/1/s/checker/tests/nullness/Issue1847B.java, /__w/1/s/checker/tests/nullness/ImplementInterface.java, /__w/1/s/checker/tests/nullness/RawInt2.java, /__w/1/s/checker/tests/nullness/Issue961.java, /__w/1/s/checker/tests/nullness/AssertAfter2.java, /__w/1/s/checker/tests/nullness/Issue764.java, /__w/1/s/checker/tests/nullness/StringTernaryConcat.java, /__w/1/s/checker/tests/nullness/FlowCompound.java, /__w/1/s/checker/tests/nullness/Issue289.java, /__w/1/s/checker/tests/nullness/WeakHasherMap.java, /__w/1/s/checker/tests/nullness/SetIteratorTest.java, /__w/1/s/checker/tests/nullness/ArrayCreationNullable.java, /__w/1/s/checker/tests/nullness/RawTypesAssignment.java, /__w/1/s/checker/tests/nullness/AnonymousSkipDefs.java, /__w/1/s/checker/tests/nullness/Issue2565.java, /__w/1/s/checker/tests/nullness/GetConstantStr.java, /__w/1/s/checker/tests/nullness/TestFromPullRequest880.java, /__w/1/s/checker/tests/nullness/ConditionalOr.java, /__w/1/s/checker/tests/nullness/Issue415.java, /__w/1/s/checker/tests/nullness/FlowConstructor.java, /__w/1/s/checker/tests/nullness/ReferencesDefaults.java, /__w/1/s/checker/tests/nullness/MissingBoundAnnotations.java, /__w/1/s/checker/tests/nullness/FinalVar2.java, /__w/1/s/checker/tests/nullness/Issue296.java, /__w/1/s/checker/tests/nullness/Issue419.java, /__w/1/s/checker/tests/nullness/Issue355.java, /__w/1/s/checker/tests/nullness/AssertIfFalseTest2.java, /__w/1/s/checker/tests/nullness/Issue765.java, /__w/1/s/checker/tests/nullness/NonNullIteratorNext.java, /__w/1/s/checker/tests/nullness/OverrideANNA.java, /__w/1/s/checker/tests/nullness/KeyForPropagation.java, /__w/1/s/checker/tests/nullness/FlowSelf.java, /__w/1/s/checker/tests/nullness/SwitchTest.java, /__w/1/s/checker/tests/nullness/SelfAssignment.java, /__w/1/s/checker/tests/nullness/Issue376.java, /__w/1/s/checker/tests/nullness/AssertIfChecked.java, /__w/1/s/checker/tests/nullness/Issue372.java, /__w/1/s/checker/tests/nullness/Issue160.java, /__w/1/s/checker/tests/nullness/Listener.java, /__w/1/s/checker/tests/nullness/KeyForPostcondition.java, /__w/1/s/checker/tests/nullness/Issue577.java, /__w/1/s/checker/tests/nullness/FinalVar3.java, /__w/1/s/checker/tests/nullness/FlowLoop.java, /__w/1/s/checker/tests/nullness/Issue295.java, /__w/1/s/checker/tests/nullness/AnnotatedJdkTest.java, /__w/1/s/checker/tests/nullness/Issue2619b.java, /__w/1/s/checker/tests/nullness/ChicoryPremain.java, /__w/1/s/checker/tests/nullness/Issue811.java, /__w/1/s/checker/tests/nullness/Asserts.java, /__w/1/s/checker/tests/nullness/EnumStaticBlock.java, /__w/1/s/checker/tests/nullness/ParameterExpression.java, /__w/1/s/checker/tests/nullness/NonEmptyCollection.java, /__w/1/s/checker/tests/nullness/TestPolyNull.java, /__w/1/s/checker/tests/nullness/AssertNonNullTest.java, /__w/1/s/checker/tests/nullness/Issue411.java, /__w/1/s/checker/tests/nullness/MisuseProperties.java, /__w/1/s/checker/tests/nullness/Issue578.java, /__w/1/s/checker/tests/nullness/KeyForShadowing.java, /__w/1/s/checker/tests/nullness/Issue266.java, /__w/1/s/checker/tests/nullness/UnusedOnClass.java, /__w/1/s/checker/tests/nullness/PolymorphismArrays.java, /__w/1/s/checker/tests/nullness/SuperCall.java, /__w/1/s/checker/tests/nullness/FlowExpressionParsingBug.java, /__w/1/s/checker/tests/nullness/Issue500.java, /__w/1/s/checker/tests/nullness/Issue427.java, /__w/1/s/checker/tests/nullness/ArrayInitBug.java, /__w/1/s/checker/tests/nullness/Issue2407.java, /__w/1/s/checker/tests/nullness/Initializer.java, /__w/1/s/checker/tests/nullness/Issue1555.java, /__w/1/s/checker/tests/nullness/Issue1406.java, /__w/1/s/checker/tests/nullness/Issue759.java, /__w/1/s/checker/tests/nullness/Issue579Error.java, /__w/1/s/checker/tests/nullness/ArrayLazyNN.java, /__w/1/s/checker/tests/nullness/OverrideGenerics.java, /__w/1/s/checker/tests/nullness/AliasedAnnotations.java, /__w/1/s/checker/tests/nullness/ThreadLocalTest.java, /__w/1/s/checker/tests/nullness/Issue414.java, /__w/1/s/checker/tests/nullness/StaticInitializer.java, /__w/1/s/checker/tests/nullness/DefaultsNullness.java, /__w/1/s/checker/tests/nullness/Issue400.java, /__w/1/s/checker/tests/nullness/Issue2865.java, /__w/1/s/checker/tests/nullness/Bug103.java, /__w/1/s/checker/tests/nullness/InitThrows.java, /__w/1/s/checker/tests/nullness/DefaultAnnotation.java, /__w/1/s/checker/tests/nullness/Issue261.java, /__w/1/s/checker/tests/nullness/WildcardSubtype2.java, /__w/1/s/checker/tests/nullness/FlowAssignment.java, /__w/1/s/checker/tests/nullness/Issue370.java, /__w/1/s/checker/tests/nullness/KeyForChecked.java, /__w/1/s/checker/tests/nullness/Issue2721.java, /__w/1/s/checker/tests/nullness/ExpressionsNullness.java, /__w/1/s/checker/tests/nullness/JavaCopFlow.java, /__w/1/s/checker/tests/nullness/BinarySearch.java, /__w/1/s/checker/tests/nullness/FinalVar.java, /__w/1/s/checker/tests/nullness/AssertWithStatic.java, /__w/1/s/checker/tests/nullness/Issue2247.java, /__w/1/s/checker/tests/nullness/KeyForSubtyping.java, /__w/1/s/checker/tests/nullness/TestInfer.java, /__w/1/s/checker/tests/nullness/DefaultFlow.java, /__w/1/s/checker/tests/nullness/TryCatch.java, /__w/1/s/checker/tests/nullness/StaticInLoop.java, /__w/1/s/checker/tests/nullness/Issue2587.java, /__w/1/s/checker/tests/nullness/Issue388.java, /__w/1/s/checker/tests/nullness/InvariantTypes.java, /__w/1/s/checker/tests/nullness/Issue1981.java, /__w/1/s/checker/tests/nullness/Issue408.java, /__w/1/s/checker/tests/nullness/EnsuresNonNullIfTest4.java, /__w/1/s/checker/tests/nullness/ArrayNew.java, /__w/1/s/checker/tests/nullness/Issue266a.java, /__w/1/s/checker/tests/nullness/KeyForLub.java, /__w/1/s/checker/tests/nullness/DefaultInterface.java, /__w/1/s/checker/tests/nullness/Issue345.java, /__w/1/s/checker/tests/nullness/Unboxing.java, /__w/1/s/checker/tests/nullness/ExtendsArrayList.java, /__w/1/s/checker/tests/nullness/Exceptions.java, /__w/1/s/checker/tests/nullness/FlowCompoundConcatenation.java, /__w/1/s/checker/tests/nullness/Issue738.java, /__w/1/s/checker/tests/nullness/Issue1147.java, /__w/1/s/checker/tests/nullness/InferNullType.java, /__w/1/s/checker/tests/nullness/Issue67.java, /__w/1/s/checker/tests/nullness/OverrideNNOE.java, /__w/1/s/checker/tests/nullness/TryWithResources.java, /__w/1/s/checker/tests/nullness/Issue2564.java, /__w/1/s/checker/tests/nullness/NegatingConditionalNullness.java, /__w/1/s/checker/tests/nullness/CastsNullness.java, /__w/1/s/checker/tests/nullness/Simple2.java, /__w/1/s/checker/tests/nullness/CompoundAssign.java, /__w/1/s/checker/tests/nullness/NNOEStaticFields.java
Testsuite: tests.NullnessFbcTest
```

Are these expected?

Reply:
> The test fails:
> 
> ```
> tests.NullnessFbcTest > run[nullness] FAILED
>     java.lang.AssertionError: 790 out of 790 expected diagnostics were found.
>     3 unexpected diagnostics were found:
>     Issue2432b.java:12: error: (assignment.type.incompatible)
>     Issue2432b.java:19: error: (assignment.type.incompatible)
>     Issue2432b.java:35: error: (new.array.type.invalid)
>     While type-checking /__w/1/s/checker/tests/nullness/MultiAnnotations.java, /__w/1/s/checker/tests/nullness/FlowNullness.java, /__w/1/s/checker/tests/nullness/Issue2470.java, /__w/1/s/checker/tests/nullness/AssertIfFalseTest.java, /__w/1/s/checker/tests/nullness/TestValOf.java, /__w/1/s/checker/tests/nullness/SequenceAndIndices.java, /__w/1/s/checker/tests/nullness/ThisTest.java, /__w/1/s/checker/tests/nullness/ReadyReadLine.java, /__w/1/s/checker/tests/nullness/AssertInStatic.java, /__w/1/s/checker/tests/nullness/JPanelTest.java, /__w/1/s/checker/tests/nullness/RawTypesBounded.java, /__w/1/s/checker/tests/nullness/SuppressWarningsTest.java, /__w/1/s/checker/tests/nullness/Issue2888.java, /__w/1/s/checker/tests/nullness/JavaCopExplosion.java, /__w/1/s/checker/tests/nullness/SuperConstructorInit.java, /__w/1/s/checker/tests/nullness/GeneralATFStore.java, /__w/1/s/checker/tests/nullness/EqualToNullness.java, /__w/1/s/checker/tests/nullness/ForEachMin.java, /__w/1/s/checker/tests/nullness/Issue328.java, /__w/1/s/checker/tests/nullness/Bug102.java, /__w/1/s/checker/tests/nullness/BoxingNullness.java, /__w/1/s/checker/tests/nullness/AssertIfClient.java, /__w/1/s/checker/tests/nullness/Issue293.java, /__w/1/s/checker/tests/nullness/Issue1797.java, /__w/1/s/checker/tests/nullness/Imports1.java, /__w/1/s/checker/tests/nullness/JavaExprContext.java, /__w/1/s/checker/tests/nullness/TernaryNullness.java, /__w/1/s/checker/tests/nullness/Issue2048.java, /__w/1/s/checker/tests/nullness/AssertParameterNullness.java, /__w/1/s/checker/tests/nullness/RawField.java, /__w/1/s/checker/tests/nullness/AssertAfter.java, /__w/1/s/checker/tests/nullness/Issue1983.java, /__w/1/s/checker/tests/nullness/Issue2619.java, /__w/1/s/checker/tests/nullness/LogicOperations.java, /__w/1/s/checker/tests/nullness/NullnessFieldInvar.java, /__w/1/s/checker/tests/nullness/KeyForFlow.java, /__w/1/s/checker/tests/nullness/KeyForSubst.java, /__w/1/s/checker/tests/nullness/Issue580.java, /__w/1/s/checker/tests/nullness/AssertIfNonNullTest.java, /__w/1/s/checker/tests/nullness/EnsuresNonNullIfInheritedTest.java, /__w/1/s/checker/tests/nullness/Issue1922.java, /__w/1/s/checker/tests/nullness/Issue308.java, /__w/1/s/checker/tests/nullness/Issue2052.java, /__w/1/s/checker/tests/nullness/DaikonTests.java, /__w/1/s/checker/tests/nullness/PureTest.java, /__w/1/s/checker/tests/nullness/TernaryNested.java, /__w/1/s/checker/tests/nullness/Synchronization.java, /__w/1/s/checker/tests/nullness/Marino.java, /__w/1/s/checker/tests/nullness/FlowField.java, /__w/1/s/checker/tests/nullness/PrimitivesNullness.java, /__w/1/s/checker/tests/nullness/UnannoPrimitives.java, /__w/1/s/checker/tests/nullness/FlowNonThis.java, /__w/1/s/checker/tests/nullness/PrivateMethodUnknownInit.java, /__w/1/s/checker/tests/nullness/Issue1949.java, /__w/1/s/checker/tests/nullness/NewObjectNonNull.java, /__w/1/s/checker/tests/nullness/InferListParam.java, /__w/1/s/checker/tests/nullness/HasInnerClass.java, /__w/1/s/checker/tests/nullness/LogRecordTest.java, /__w/1/s/checker/tests/nullness/MultipleErrors.java, /__w/1/s/checker/tests/nullness/EnsuresNonNullIfTestSimple.java, /__w/1/s/checker/tests/nullness/Issue331.java, /__w/1/s/checker/tests/nullness/GetProperty.java, /__w/1/s/checker/tests/nullness/GetRefArg.java, /__w/1/s/checker/tests/nullness/AssertTwice.java, /__w/1/s/checker/tests/nullness/Issue1096.java, /__w/1/s/checker/tests/nullness/Conditions.java, /__w/1/s/checker/tests/nullness/KeyForLocalVariable.java, /__w/1/s/checker/tests/nullness/TestAssignment.java, /__w/1/s/checker/tests/nullness/Throwing.java, /__w/1/s/checker/tests/nullness/Issue273.java, /__w/1/s/checker/tests/nullness/DotClass.java, /__w/1/s/checker/tests/nullness/FlowConditions.java, /__w/1/s/checker/tests/nullness/KeyForIssue328.java, /__w/1/s/checker/tests/nullness/VoidUse.java, /__w/1/s/checker/tests/nullness/MethodInvocation.java, /__w/1/s/checker/tests/nullness/Aliasing.java, /__w/1/s/checker/tests/nullness/FinalFields.java, /__w/1/s/checker/tests/nullness/KeyForDiamond.java, /__w/1/s/checker/tests/nullness/LubTest.java, /__w/1/s/checker/tests/nullness/WeakIdentityPair.java, /__w/1/s/checker/tests/nullness/Issue2031.java, /__w/1/s/checker/tests/nullness/OverrideANNA3.java, /__w/1/s/checker/tests/nullness/ChainAssignment.java, /__w/1/s/checker/tests/nullness/VarargsNullness.java, /__w/1/s/checker/tests/nullness/CastTypeVariable.java, /__w/1/s/checker/tests/nullness/HierarchicalInit.java, /__w/1/s/checker/tests/nullness/RequiresNonNullTest.java, /__w/1/s/checker/tests/nullness/NewNullable.java, /__w/1/s/checker/tests/nullness/Issue906.java, /__w/1/s/checker/tests/nullness/Issue1027.java, /__w/1/s/checker/tests/nullness/RawParameter.java, /__w/1/s/checker/tests/nullness/UnboxConditions.java, /__w/1/s/checker/tests/nullness/ThisIsNN.java, /__w/1/s/checker/tests/nullness/UnexpectedRaw.java, /__w/1/s/checker/tests/nullness/Issue752.java, /__w/1/s/checker/tests/nullness/SuppressDeprecation.java, /__w/1/s/checker/tests/nullness/Issue563.java, /__w/1/s/checker/tests/nullness/InfiniteLoopIsSameType.java, /__w/1/s/checker/tests/nullness/ValidType.java, /__w/1/s/checker/tests/nullness/RawTypesUses.java, /__w/1/s/checker/tests/nullness/Issue1628.java, /__w/1/s/checker/tests/nullness/FullyQualifiedAnnotation.java, /__w/1/s/checker/tests/nullness/RawAndPrimitive.java, /__w/1/s/checker/tests/nullness/Issue989.java, /__w/1/s/checker/tests/nullness/TwoStaticInitBlocks.java, /__w/1/s/checker/tests/nullness/MapGetNullable.java, /__w/1/s/checker/tests/nullness/InitSuppressWarnings.java, /__w/1/s/checker/tests/nullness/AssertAfterChecked.java, /__w/1/s/checker/tests/nullness/ToArrayNullness.java, /__w/1/s/checker/tests/nullness/FlowNegation.java, /__w/1/s/checker/tests/nullness/OverrideNNOE2.java, /__w/1/s/checker/tests/nullness/PackageDecl.java, /__w/1/s/checker/tests/nullness/Issue369.java, /__w/1/s/checker/tests/nullness/Issue1522.java, /__w/1/s/checker/tests/nullness/KeyForPolymorphism.java, /__w/1/s/checker/tests/nullness/NonNullInitialization.java, /__w/1/s/checker/tests/nullness/ThisLiteral.java, /__w/1/s/checker/tests/nullness/OverrideANNA2.java, /__w/1/s/checker/tests/nullness/MethodTypeVars4.java, /__w/1/s/checker/tests/nullness/ArrayArgs.java, /__w/1/s/checker/tests/nullness/LazyInitialization.java, /__w/1/s/checker/tests/nullness/EnsuresNonNullIfTest2.java, /__w/1/s/checker/tests/nullness/Wildcards.java, /__w/1/s/checker/tests/nullness/Wellformed.java, /__w/1/s/checker/tests/nullness/Imports2.java, /__w/1/s/checker/tests/nullness/VarInfoName.java, /__w/1/s/checker/tests/nullness/MultiConstructorInit.java, /__w/1/s/checker/tests/nullness/Issue1712.java, /__w/1/s/checker/tests/nullness/Issue471.java, /__w/1/s/checker/tests/nullness/InitializedField.java, /__w/1/s/checker/tests/nullness/FlowConstructor2.java, /__w/1/s/checker/tests/nullness/AssertNullable.java, /__w/1/s/checker/tests/nullness/Issue1102.java, /__w/1/s/checker/tests/nullness/Raw2.java, /__w/1/s/checker/tests/nullness/Enums.java, /__w/1/s/checker/tests/nullness/Issue1307.java, /__w/1/s/checker/tests/nullness/ArrayAssignmentFlow.java, /__w/1/s/checker/tests/nullness/IteratorEarlyExit.java, /__w/1/s/checker/tests/nullness/KeyFors.java, /__w/1/s/checker/tests/nullness/BinaryOp.java, /__w/1/s/checker/tests/nullness/SAMLineParser.java, /__w/1/s/checker/tests/nullness/PostconditionBug.java, /__w/1/s/checker/tests/nullness/MonotonicNonNullTest.java, /__w/1/s/checker/tests/nullness/Widening.java, /__w/1/s/checker/tests/nullness/ArrayRefs.java, /__w/1/s/checker/tests/nullness/RefineArray.java, /__w/1/s/checker/tests/nullness/Issue2171.java, /__w/1/s/checker/tests/nullness/Issue2432.java, /__w/1/s/checker/tests/nullness/KeyForValidation.java, /__w/1/s/checker/tests/nullness/InitializationAssertionFailure.java, /__w/1/s/checker/tests/nullness/JavaCopRandomTests.java, /__w/1/s/checker/tests/nullness/ConditionalNullness.java, /__w/1/s/checker/tests/nullness/NullableArrays.java, /__w/1/s/checker/tests/nullness/Lazy.java, /__w/1/s/checker/tests/nullness/Issue829.java, /__w/1/s/checker/tests/nullness/WeakRef.java, /__w/1/s/checker/tests/nullness/RawInt.java, /__w/1/s/checker/tests/nullness/NNOEMoreTests.java, /__w/1/s/checker/tests/nullness/DefaultLoops.java, /__w/1/s/checker/tests/nullness/Issue391.java, /__w/1/s/checker/tests/nullness/UnannoPrimitivesDefaults.java, /__w/1/s/checker/tests/nullness/WhileTest.java, /__w/1/s/checker/tests/nullness/Polymorphism.java, /__w/1/s/checker/tests/nullness/Issue672.java, /__w/1/s/checker/tests/nullness/FlowInitialization.java, /__w/1/s/checker/tests/nullness/MyException.java, /__w/1/s/checker/tests/nullness/FieldInit.java, /__w/1/s/checker/tests/nullness/WildcardSubtype.java, /__w/1/s/checker/tests/nullness/Issue1059.java, /__w/1/s/checker/tests/nullness/TypeVarPrimitivesNullness.java, /__w/1/s/checker/tests/nullness/CopyOfArray.java, /__w/1/s/checker/tests/nullness/Issue1046Java7.java, /__w/1/s/checker/tests/nullness/RawTypesNullness.java, /__w/1/s/checker/tests/nullness/FindBugs.java, /__w/1/s/checker/tests/nullness/Issue2432b.java, /__w/1/s/checker/tests/nullness/ControlFlow.java, /__w/1/s/checker/tests/nullness/InferTypeArgsCondtionalExpression.java, /__w/1/s/checker/tests/nullness/RawMethodInvocation.java, /__w/1/s/checker/tests/nullness/ExplictTypeVarAnnos.java, /__w/1/s/checker/tests/nullness/ExceptionParam.java, /__w/1/s/checker/tests/nullness/Issue1847.java, /__w/1/s/checker/tests/nullness/Issue338.java, /__w/1/s/checker/tests/nullness/CallSuper.java, /__w/1/s/checker/tests/nullness/AssignmentDuringInitialization.java, /__w/1/s/checker/tests/nullness/Issue1847B.java, /__w/1/s/checker/tests/nullness/ImplementInterface.java, /__w/1/s/checker/tests/nullness/RawInt2.java, /__w/1/s/checker/tests/nullness/Issue961.java, /__w/1/s/checker/tests/nullness/AssertAfter2.java, /__w/1/s/checker/tests/nullness/Issue764.java, /__w/1/s/checker/tests/nullness/StringTernaryConcat.java, /__w/1/s/checker/tests/nullness/FlowCompound.java, /__w/1/s/checker/tests/nullness/Issue289.java, /__w/1/s/checker/tests/nullness/WeakHasherMap.java, /__w/1/s/checker/tests/nullness/SetIteratorTest.java, /__w/1/s/checker/tests/nullness/ArrayCreationNullable.java, /__w/1/s/checker/tests/nullness/RawTypesAssignment.java, /__w/1/s/checker/tests/nullness/AnonymousSkipDefs.java, /__w/1/s/checker/tests/nullness/Issue2565.java, /__w/1/s/checker/tests/nullness/GetConstantStr.java, /__w/1/s/checker/tests/nullness/TestFromPullRequest880.java, /__w/1/s/checker/tests/nullness/ConditionalOr.java, /__w/1/s/checker/tests/nullness/Issue415.java, /__w/1/s/checker/tests/nullness/FlowConstructor.java, /__w/1/s/checker/tests/nullness/ReferencesDefaults.java, /__w/1/s/checker/tests/nullness/MissingBoundAnnotations.java, /__w/1/s/checker/tests/nullness/FinalVar2.java, /__w/1/s/checker/tests/nullness/Issue296.java, /__w/1/s/checker/tests/nullness/Issue419.java, /__w/1/s/checker/tests/nullness/Issue355.java, /__w/1/s/checker/tests/nullness/AssertIfFalseTest2.java, /__w/1/s/checker/tests/nullness/Issue765.java, /__w/1/s/checker/tests/nullness/NonNullIteratorNext.java, /__w/1/s/checker/tests/nullness/OverrideANNA.java, /__w/1/s/checker/tests/nullness/KeyForPropagation.java, /__w/1/s/checker/tests/nullness/FlowSelf.java, /__w/1/s/checker/tests/nullness/SwitchTest.java, /__w/1/s/checker/tests/nullness/SelfAssignment.java, /__w/1/s/checker/tests/nullness/Issue376.java, /__w/1/s/checker/tests/nullness/AssertIfChecked.java, /__w/1/s/checker/tests/nullness/Issue372.java, /__w/1/s/checker/tests/nullness/Issue160.java, /__w/1/s/checker/tests/nullness/Listener.java, /__w/1/s/checker/tests/nullness/KeyForPostcondition.java, /__w/1/s/checker/tests/nullness/Issue577.java, /__w/1/s/checker/tests/nullness/FinalVar3.java, /__w/1/s/checker/tests/nullness/FlowLoop.java, /__w/1/s/checker/tests/nullness/Issue295.java, /__w/1/s/checker/tests/nullness/AnnotatedJdkTest.java, /__w/1/s/checker/tests/nullness/Issue2619b.java, /__w/1/s/checker/tests/nullness/ChicoryPremain.java, /__w/1/s/checker/tests/nullness/Issue811.java, /__w/1/s/checker/tests/nullness/Asserts.java, /__w/1/s/checker/tests/nullness/EnumStaticBlock.java, /__w/1/s/checker/tests/nullness/ParameterExpression.java, /__w/1/s/checker/tests/nullness/NonEmptyCollection.java, /__w/1/s/checker/tests/nullness/TestPolyNull.java, /__w/1/s/checker/tests/nullness/AssertNonNullTest.java, /__w/1/s/checker/tests/nullness/Issue411.java, /__w/1/s/checker/tests/nullness/MisuseProperties.java, /__w/1/s/checker/tests/nullness/Issue578.java, /__w/1/s/checker/tests/nullness/KeyForShadowing.java, /__w/1/s/checker/tests/nullness/Issue266.java, /__w/1/s/checker/tests/nullness/UnusedOnClass.java, /__w/1/s/checker/tests/nullness/PolymorphismArrays.java, /__w/1/s/checker/tests/nullness/SuperCall.java, /__w/1/s/checker/tests/nullness/FlowExpressionParsingBug.java, /__w/1/s/checker/tests/nullness/Issue500.java, /__w/1/s/checker/tests/nullness/Issue427.java, /__w/1/s/checker/tests/nullness/ArrayInitBug.java, /__w/1/s/checker/tests/nullness/Issue2407.java, /__w/1/s/checker/tests/nullness/Initializer.java, /__w/1/s/checker/tests/nullness/Issue1555.java, /__w/1/s/checker/tests/nullness/Issue1406.java, /__w/1/s/checker/tests/nullness/Issue759.java, /__w/1/s/checker/tests/nullness/Issue579Error.java, /__w/1/s/checker/tests/nullness/ArrayLazyNN.java, /__w/1/s/checker/tests/nullness/OverrideGenerics.java, /__w/1/s/checker/tests/nullness/AliasedAnnotations.java, /__w/1/s/checker/tests/nullness/ThreadLocalTest.java, /__w/1/s/checker/tests/nullness/Issue414.java, /__w/1/s/checker/tests/nullness/StaticInitializer.java, /__w/1/s/checker/tests/nullness/DefaultsNullness.java, /__w/1/s/checker/tests/nullness/Issue400.java, /__w/1/s/checker/tests/nullness/Issue2865.java, /__w/1/s/checker/tests/nullness/Bug103.java, /__w/1/s/checker/tests/nullness/InitThrows.java, /__w/1/s/checker/tests/nullness/DefaultAnnotation.java, /__w/1/s/checker/tests/nullness/Issue261.java, /__w/1/s/checker/tests/nullness/WildcardSubtype2.java, /__w/1/s/checker/tests/nullness/FlowAssignment.java, /__w/1/s/checker/tests/nullness/Issue370.java, /__w/1/s/checker/tests/nullness/KeyForChecked.java, /__w/1/s/checker/tests/nullness/Issue2721.java, /__w/1/s/checker/tests/nullness/ExpressionsNullness.java, /__w/1/s/checker/tests/nullness/JavaCopFlow.java, /__w/1/s/checker/tests/nullness/BinarySearch.java, /__w/1/s/checker/tests/nullness/FinalVar.java, /__w/1/s/checker/tests/nullness/AssertWithStatic.java, /__w/1/s/checker/tests/nullness/Issue2247.java, /__w/1/s/checker/tests/nullness/KeyForSubtyping.java, /__w/1/s/checker/tests/nullness/TestInfer.java, /__w/1/s/checker/tests/nullness/DefaultFlow.java, /__w/1/s/checker/tests/nullness/TryCatch.java, /__w/1/s/checker/tests/nullness/StaticInLoop.java, /__w/1/s/checker/tests/nullness/Issue2587.java, /__w/1/s/checker/tests/nullness/Issue388.java, /__w/1/s/checker/tests/nullness/InvariantTypes.java, /__w/1/s/checker/tests/nullness/Issue1981.java, /__w/1/s/checker/tests/nullness/Issue408.java, /__w/1/s/checker/tests/nullness/EnsuresNonNullIfTest4.java, /__w/1/s/checker/tests/nullness/ArrayNew.java, /__w/1/s/checker/tests/nullness/Issue266a.java, /__w/1/s/checker/tests/nullness/KeyForLub.java, /__w/1/s/checker/tests/nullness/DefaultInterface.java, /__w/1/s/checker/tests/nullness/Issue345.java, /__w/1/s/checker/tests/nullness/Unboxing.java, /__w/1/s/checker/tests/nullness/ExtendsArrayList.java, /__w/1/s/checker/tests/nullness/Exceptions.java, /__w/1/s/checker/tests/nullness/FlowCompoundConcatenation.java, /__w/1/s/checker/tests/nullness/Issue738.java, /__w/1/s/checker/tests/nullness/Issue1147.java, /__w/1/s/checker/tests/nullness/InferNullType.java, /__w/1/s/checker/tests/nullness/Issue67.java, /__w/1/s/checker/tests/nullness/OverrideNNOE.java, /__w/1/s/checker/tests/nullness/TryWithResources.java, /__w/1/s/checker/tests/nullness/Issue2564.java, /__w/1/s/checker/tests/nullness/NegatingConditionalNullness.java, /__w/1/s/checker/tests/nullness/CastsNullness.java, /__w/1/s/checker/tests/nullness/Simple2.java, /__w/1/s/checker/tests/nullness/CompoundAssign.java, /__w/1/s/checker/tests/nullness/NNOEStaticFields.java
> Testsuite: tests.NullnessFbcTest
> ```
> 
> Are these expected?

Line 12 and 19 are expected, but 35 is not. I guess my way is not secure, maybe just using null as return value.